scripts/updategoogle.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+
+import json
+import sys
+
+from collections import OrderedDict
+
+def convert(text):
+    patterns = list()
+    for domain in text.split():
+        patterns.append("https://www" + domain + "/*")
+        patterns.append("http://www" + domain + "/*")
+    return patterns
+
+def update_manifest(supported_domains_path, manifest_path):
+    with open(manifest_path, 'r') as f:
+        manifest = json.load(f, object_pairs_hook=OrderedDict)
+
+    with open(supported_domains_path, 'r+') as f:
```

Comment:
maybe add a comment to explain this as well

Reply:
Good call. This seems sloppy, that we read one thing from and then write another thing to the same file inside the same function. Should probably have the reading and the writing happen independently. The calling script can pass three parameters instead of two, and if we want to reuse a temporary file, two of the arguments would be the same value. Maybe we'll fix this if we ever come back to do something else here.